pymc3/distributions/timeseries.py

```diff
@@ -156,18 +158,39 @@ def logp(self, value):
         -------
         TensorVariable
         """
+
+        if self.rho.ndim > 2:
+            # Assumes a matrix of cross-series coefficients
+            # for each time lag
+            bcast_fn = at.dot
+            if self.constant:
+                raise NotImplementedError(
+                    "Cross-series lags not currently supported with constant=True"
+                )
+        else:
+            bcast_fn = at.mul
+
         if self.constant:
```

Comment:
@almostmeenal The reason I paused on this PR was because I couldn't quite figure out exactly what the `self.contant` option was implementing. I essentially put in what looked to me like the broadcasted equivalent of it, but I'd like to figure it out some more.